src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -42,4 +46,8 @@ public Date getDate() {
         return date;
     }
 
+    public LocalDate getLocalDate() {
+        return localDate;
```

Comment:
I think this can be covered in the test case too

Reply:
Thanks! Will implement this in the test case.